packages/react-native-aztec/example/editor.js

```diff
@@ -1,24 +1,24 @@
 import React, { Component } from 'react';
 import {StyleSheet, Button, View} from 'react-native';
-import AztecView from 'react-native-aztec'
+import AztecView from '@wordpress/react-native-aztec'
 
 const _minHeight = 100;
 
 export default class Editor extends Component {
     constructor(props) {
         super(props);
-        this.onFormatPress = this.onFormatPress.bind(this) 
+        this.onFormatPress = this.onFormatPress.bind(this)
```

Comment:
Looks like my IDE cleaned up some extra spaces here (it doesn't look like `prettier` ran)